CHANGELOG.md

```diff
@@ -5,10 +5,78 @@ we recommend most users use the latest `master` branch of Teku.
 
 ## Upcoming Breaking Changes
 
-- REST API endpoints will be updated to match emerging standards in a future release.
+- REST API endpoints will be updated to match emerging standards in a future release. Current replacements:
```

Comment:
this doesn't really suggest that we're planning on deleting the old ones soon...

`Replacements have been added, endpoints that have been superseded will be removed in a future release`

Not sure thats a lot better...

Reply:
Tweaked it a bit.  Still not great but will have to do.